mlflow/tensorflow.py

```diff
@@ -912,7 +912,15 @@ def train(original, self, *args, **kwargs):
             try_mlflow_log(mlflow.log_param, "max_steps", kwargs["max_steps"])
 
         result = original(self, *args, **kwargs)
-
+        # Logs Tensorboard to artifacts.
+        for file in os.listdir(self.model_dir):
```

Comment:
can we handle the case where `self.model_dir` does not exist?

Reply:
Done.